### ISSUE TITLE
change the size of /boot to 500mb + current default layout /boot size is nearly full with linux and linux lts kernels installed.

### ISSUE THREAD

**Issue by mhussaincov93** [archlinux/archinstall#913]
hi,
would this be possible?
on my system with the default layout, having the linux and the linux-kernel installed as a fallback lieves 77m 
Majid

**Comment by dylanmtaylor** [comment]
I believe this is already possible as long as you install the system with a custom partition layout using a configuration file.

**Comment by Torxed** [comment]
We created a `/boot` partition with a size of `513 MiB` by default.
The only time we do not do this, is if you are dual booting Windows. In which case Windows creates a roughly `90 MiB` small `/boot` partition. There is sadly nothing we can do about that and using a re-size tool before installing arch is required.

It would be hard for us to reliably try to automate this, as data loss could be an issue.

**Comment by f35f0ef9d0e827dae86552d3899f78fc** [comment]
I am not dual booting, and my most recent install from about 3 weeks ago has a 200 MB /boot partition, which leaves only 55MB free with two kernels installed. It is not enough space to add a third, or even enough temporary space to swap one of the two kernels for a different one.

Are you sure it's not this [203 MiB value](https://github.com/archlinux/archinstall/blob/f107104103b74ac065a56bd94a0878f0d263e177/archinstall/lib/interactions/disk_conf.py#L187)? I could be wrong. Honest question.

I, for one, hate partitions. Waste of space and management. Besides that, why is it so small in the first place? Thumb drive installations? Thumb drives are huge today. Isn't it an easy to change a number value somewhere?

**Comment by lucidcocoon** [comment]
I believe I stumbled into this issue today installing Arch on my Libreboot T440p with a 2TB SSD. Using the "automatic" BTRFS partitioning scheme gave me a 212MB FAT32 boot partition which was insufficient when selecting the Linux and LTS kernels in the install. The installer crashed complaining it ran out of disk space.

I had to delete both partitions and use the manual partitioning option to resize them. I manually created a 1GB FAT32 boot partition (probably more than necessary but I thought it a safe value and an extra 0.5GB isn't a lot in the grand scheme of things) and a BTRFS partition with the remainder, manually copying the subvolume layout the "automatic" method created. 

My Arch install is now up and running without any errors after that.
